src/commands/bash_hook.rs

```diff
@@ -197,6 +230,34 @@ pub fn execute(matches: &ArgMatches) -> Result<()> {
         )
         .replace("___SENTRY_LOG_FILE___", &log.display().to_string());
 
+    let mut hook_tags = "".to_string();
+    for tag in tags {
+        hook_tags.push_str(&"-t ");
+        hook_tags.push_str(format!("\"{}\"", tag.to_string()));
+    }
+    if hook_tags.is_empty() {
```

Comment:
If `hook_tags` is indeed empty, replacing the placeholder with `&""` will be effectively the same as `&hook_tags.to_string()` :)

Reply:
Both comments are also applicable to `contexts`